src/tested/streetmix-utils.js

```diff
@@ -51,10 +51,21 @@ module.exports.streetmixAPIToUser = streetmixAPIToUser;
 function calcStreetWidth (segments) { // eslint-disable-line no-unused-vars
   var cumulativeWidthInMeters = 0;
   segments.forEach((currentSegment) => {
-    const segmentWidthInFeet = currentSegment.width;
-    const segmentWidthInMeters = segmentWidthInFeet * 0.3048;
-    cumulativeWidthInMeters = cumulativeWidthInMeters + segmentWidthInMeters;
+    cumulativeWidthInMeters = cumulativeWidthInMeters + currentSegment.width;
   });
   return cumulativeWidthInMeters;
 }
 module.exports.calcStreetWidth = calcStreetWidth;
+
+// convert all feet values to meters for schemaVersion < 30
+function convertStreetValues (streetData) {
```

Comment:
Thanks @Algorush , I really like this approach to modify the widths "in situ" only if needed instead of distributing that logic elsewhere in the app.